problog/tasks/explain.py

```diff
@@ -54,11 +55,11 @@ def main(argv):
         explanation = []
         results = cnf.evaluate(explain=explanation)
 
+        result["SUCCESS"] = True
+        result["program"] = program
+        result["proofs"] = explanation
+        result["results"] = results
```

Comment:
result["results"] was exclusive to the else branch while result["probabilities"] to the if branch. Now not anymore, does that matter?

Reply:
I'm not sure: it just passes more information than before to the webserver, which I would expect to be able to handle this, but maybe we should indeed try it out.

Reply:
It is probably fine.